src/claimApplication/tdiOverride.js

```diff
@@ -11,5 +12,11 @@ if (document.readyState === "loading") {
 
 function executeOverride() {
   setupAnalytics();
+
+  const pageId = identifyPage();
+  if (pageId === 'other') {
+    return; // do not apply any other changes
+  }
```

Comment:
over time there will be more methods within here and more dependence on `pageId` being one thing or another, or on some list, etc.